### ISSUE TITLE
MOTS: Clients in multiplayer broken (v0.9.0)

### ISSUE THREAD

**Issue by BathySalts** [shinyquagsire23/OpenJKDF2#319]
Clients in multiplayer are unable to take damage, pickup items, or equip any weapons. This issue appears to be specific to MOTS.

https://github.com/shinyquagsire23/OpenJKDF2/assets/76634216/c1b99f9a-ac15-4f76-b75d-bcf6cd8da1ce

**Comment by BathySalts** [comment]
Same Bug Still Persists In Version 0.9.5 And 0.9.6

**Comment by shinyquagsire23** [comment]
Yeah I tracked down the weird tripmine bug recently, but this one and the weird joined-but-invisible bug have been stubborn unfortunately

**Comment by Checkmate9-5** [comment]
I noticed some additional details, hopefully useful in tracking down the bug. Every time someone joins, the host reverts back to the default loadout (saber and bryar pistol with 50 ammo), almost like the game thinks the host is joining again.
